lib/kubeclient/common.rb

```diff
@@ -310,7 +315,11 @@ def watch_pod_log(pod_name, namespace, container: nil)
     end
 
     def proxy_url(kind, name, port, namespace = '')
-      entity_name_plural = ClientMixin.pluralize_entity(kind.to_s)
+      entity_name_plural = if %w(services pods nodes).include?(kind.to_s)
```

Comment:
Previously the code was nicer
```if kind.to_s.end_with?('s')```
But this makes it clearer this is for backward comp that only handles 3 entities.